data/map.txt

```diff
@@ -265,7 +265,7 @@ system "11 Autumn Above"
 		period 14.9791
 		offset 180
 	object
-		sprite planet/gas1-b
+		sprite "planet/hot jupiter10"
```

Comment:
To be in keeping with the naming scheme of existing planets, perhaps use `gas#-hot` as the naming scheme instead of `hot jupiter#`

Reply:
Done, both here and in high DPI